src/index.html

```diff
@@ -2,13 +2,29 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="./images/logo.png" alt="Moyo"/></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#" class="nav__item is-active" target="_self">Apple</a></li>
+          <li><a href="#" class="nav__item">Samsung</a></li>
+          <li><a href="#" class="nav__item">Smartphones</a></li>
+          <li><a href="#" class="nav__item" data-qa="hover">Laptops&Computers</a></li>
+          <li><a href="#" class="nav__item">Gadgets</a></li>
+          <li><a href="#" class="nav__item">Tablets</a></li>
+          <li><a href="#" class="nav__item">Photo</a></li>
+          <li><a href="#" class="nav__item">Video</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="nav__item is-active" target="_self">
              Apple
            </a>
          </li>
          <li><a href="#" class="nav__item">Samsung</a></li>
          <li><a href="#" class="nav__item">Smartphones</a></li>
          <li>
            <a 
              href="#" 
              class="nav__item" 
              data-qa="hover"
            >Laptops&Computers
            </a>
          </li>
          <li><a href="#" class="nav__item">Gadgets</a></li>
          <li><a href="#" class="nav__item">Tablets</a></li>
          <li><a href="#" class="nav__item">Photo</a></li>
          <li><a href="#" class="nav__item">Video</a></li>
```

Reply:
better to format like this. children should be on new line with +indentation, attributes when more then 2 too.